src/Symfony/Component/Ldap/Tests/LdapClientTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Ldap\Tests;
+
+use Symfony\Component\Ldap\LdapClient;
+use Symfony\Polyfill\Php56\Php56 as p;
+
+class LdapClientTest extends \PHPUnit_Framework_TestCase
```

Comment:
You should skip the test if the extension is not enabled, which is the case in one of the two Appveyor builds (one is run with the extension, the other without).